### DISCUSSION TITLE
Please enable captions for community calls

### DISCUSSION THREAD

**Discussion by aardrian** [mdn/mdn-community#423] [Announcements]
I joined the LLM call today but there were no captions provided nor was Zoom configured to allow me to use its auto-generated caption feature.

I appreciate that when I asked in the chat members of the team looked into it, but all they could do was confirm there were no captions.

While automated captions are far from ideal, they at least would allow participation from _more_ of the community than having none. Please enable _at least_ automated captions for community calls.

Some of the irony here is that I joined the LLM community call because I was concerned about the accessibility of the content it had produced (noted in the two issues already). So now I am concerned about the accessibility of the discussions around the accessibility of the content.

<img width="289" alt="Me at 12:37pm in Zoom chat: 'Are the captions enabled for this call?' Claas Augner responding immediately by asking '@Konstantina Padadea Do we have a way to enable captions?' Konstantina saying she is looking into it and then responding at 12:41pm `Unfortunately no, just looked there is not an automated way to do so, frowny emoji.'" src="https://github.com/mdn/mdn-community/assets/1376607/96c9f5a6-c520-4e8c-92ce-fc0b11eaf943">

_Note:_ I am including the chat screen shot so you can see that MDN folks on the call were quick to respond. I appreciate they jumped on it even if the outcome was meh (and seemingly out of their control).

**Comment by caugner** [comment]
We're very sorry that we couldn't enable captions on the Community Call, but it really was out of our control. Internally, we followed up with the team that manages our Zoom accounts, and they enabled captions on the calls hosted by @couci, so we should be good in future Community Calls! 🎉

**Comment by aardrian** [comment]
I am glad to hear captions will be enabled for at least some calls. I have follow-up questions...

> they enabled captions on the calls hosted by @couci,

1. Does this mean auto-generated captions?
2. I don't know how many calls @couci hosts. Can they enable it for _all_ community calls regardless of host?

**Comment by couci** [comment]
Hey @aardrian , yes those are autogenerated captions.
Unfortunately the feature can be enabled per user and not per call, so for now AV was able to enable it for community calls I host.
The plan is that I will create those zoom rooms for the team so we can make sure that all of the future community calls are with captions.
